app/ui-react/yarn.lock

```diff
@@ -3176,10 +3176,10 @@
     universal-user-agent "^2.0.0"
     url-template "^2.0.8"
 
-"@patternfly/patternfly@2.43.1":
-  version "2.43.1"
-  resolved "https://registry.yarnpkg.com/@patternfly/patternfly/-/patternfly-2.43.1.tgz#9961e331ac78f2cf55a73527656503f5555b053e"
-  integrity sha512-dWw4FeQfLHFXva1VmH8Epusu3VX91VrOH+9FVvRDsCEcajsq5Pc5BCF+H9KyQ4eA6EfTEMsW2F0zVWD/7B4RFA==
+"@patternfly/patternfly@2.40.6", "@patternfly/patternfly@2.43.1":
+  version "2.40.6"
```

Comment:
I thought we were going to need to upgrade the Patternfly version, not downgrade. Am I missing something?

Reply:
Not touching PF for now, looks like we've gotten the version we need for this feature to work.